plugins/webapi/tools/message/diagnostic_branches.go

```diff
@@ -0,0 +1,167 @@
+package message
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/labstack/echo"
+
+	"github.com/iotaledger/goshimmer/packages/ledgerstate"
+	"github.com/iotaledger/goshimmer/packages/tangle"
+	"github.com/iotaledger/goshimmer/plugins/messagelayer"
+)
+
+// DiagnosticBranchesHandler runs the diagnostic over the Tangle.
+func DiagnosticBranchesHandler(c echo.Context) (err error) {
+	runDiagnosticBranches(c)
+	return
+}
+
+// DiagnosticLazyBookedBranchesHandler runs the diagnostic over the Tangle.
+func DiagnosticLazyBookedBranchesHandler(c echo.Context) (err error) {
+	runDiagnosticChildBranches(c, ledgerstate.LazyBookedConflictsBranchID)
+	return
+}
+
+// DiagnosticInvalidBranchesHandler runs the diagnostic over the Tangle.
+func DiagnosticInvalidBranchesHandler(c echo.Context) (err error) {
+	runDiagnosticChildBranches(c, ledgerstate.InvalidBranchID)
+	return
+}
+
+// region Analysis code implementation /////////////////////////////////////////////////////////////////////////////////
+
+func runDiagnosticBranches(c echo.Context) {
+	// write Header and table description
+	c.Response().Header().Set(echo.HeaderContentType, "text/csv")
+	c.Response().WriteHeader(http.StatusOK)
+
+	_, err := fmt.Fprintln(c.Response(), strings.Join(DiagnosticBranchesTableDescription, ","))
+	if err != nil {
+		panic(err)
+	}
+
+	messagelayer.Tangle().LedgerState.BranchDAG.ForEachBranch(func(branch ledgerstate.Branch) {
+		if branch.ID() != ledgerstate.MasterBranchID &&
+			branch.ID() != ledgerstate.InvalidBranchID &&
+			branch.ID() != ledgerstate.LazyBookedConflictsBranchID {
```

Comment:
We can handle check like this with switch cases :rocket:  
```suggestion
		switch branch.ID() {
		    case ledgerstate.MasterBranchID:
		    case ledgerstate.InvalidBranchID:
		    case ledgerstate.LazyBookedConflictsBranchID:
		        return
		    default:
		            .....
		 }
```